pkg/scaffold/olm-catalog/testdata/deploy/olm-catalog/app-operator/0.1.0/app-operator.v0.1.0.clusterserviceversion.yaml

```diff
@@ -8,12 +8,20 @@ spec:
   apiservicedefinitions: {}
   customresourcedefinitions:
     owned:
-    - kind: App
```

Comment:
If we update the test data, should we not need to adjust or add the test as well? I am not familiar with the test setup so just curious.

Reply:
The test data was updated because `customresourcedefinitions` updates were not being tested properly. Now the two `owned` CRD's in the CSV have different data, so the unit tests are more likely to pick up scaffolding errors.